src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzerTests.cs

```diff
@@ -1094,19 +1062,8 @@ End Class
                 {
                     Sources = { source },
                     AdditionalFiles = { (AdditionalFileName, additionalText) },
-                    ExpectedDiagnostics =
-                    {
-                        GetBasicRS1028ResultAt(12, 70),
-                        GetBasicRS1028ResultAt(13, 71),
-                        GetBasicRS1028ResultAt(14, 73),
-                        GetBasicRS1028ResultAt(15, 71),
-                        GetBasicRS1028ResultAt(16, 71),
-                        GetBasicRS1028ResultAt(17, 71),
-                        GetBasicRS1028ResultAt(18, 71),
-                        GetBasicRS1028ResultAt(19, 71),
-                    }
                 },
-                SolutionTransforms = { WithoutEnableReleaseTrackingWarning }
+                SolutionTransforms = { (solution, projectId) => WithoutOtherDiagnosticDescriptorRules(solution, projectId, new[] { DiagnosticDescriptorCreationAnalyzer.DiagnosticIdMustBeInSpecifiedFormatRule, DiagnosticDescriptorCreationAnalyzer.UseCategoriesFromSpecifiedRangeRule, }) },
```

Comment:
Can we avoid each test having to specify the array by creating helpers, similar to the one that previously existed?

Reply:
That's a good idea!